web-client/dist/README.md

```diff
@@ -3,7 +3,7 @@
 A very light Nimiq Proof-of-Stake client that runs in web browsers, compiled from Rust to WebAssembly.
 
 > **Note**
-> This web client is intended to be used in web browsers only, as it requires `window` to be available (no WASI support either).
+> This web client is intended to be used in web browsers only (no WASI support either). Other webworker-enabled environments are not yet targeted.
```

Comment:
Maybe _supported_ instead of _targeted_?

Reply:
Ok.